### PR TITLE
kvdb: bump to sqldb/v1.0.1

### PR DESCRIPTION
**Pull request by Roasbeef**
Post merge module clean up after
https://github.com/lightningnetwork/lnd/pull/8611.

### CODE REVIEW

**Comment by Roasbeef** [comment]
Remaining steps now:
  1. tag new `sqldb `version
  2. push that out
  3. PR to update `kvdb` to that new version
  4. tag new `kvdb` version 
  5. update top level lnd module to new `kvdb` and `sqldb` version


This PR is step 3.

**Comment by Roasbeef** [comment]
Latest commit also removes the top level replace. 

After we merge this, then we tag a new `kvdb` version, and update the top level module there.

**Review by bhandras — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -203,14 +203,6 @@ replace github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
 // allows us to specify that as an option.
 replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-display v1.30.0-hex-display
 
-// Temporary replace rule until kvdb is tagged with changes from
-// https://github.com/lightningnetwork/lnd/pull/8611.
-replace github.com/lightningnetwork/lnd/kvdb => ./kvdb
```
Comment (guggero) [inline go.mod:208]: I think the `kvdb` replace needs to remain until we also pushed a new tag of that?

**Review by guggero — CHANGES_REQUESTED** [review]

**Comment by guggero** [comment]
While we're at it, can we also bump `healthcheck` to `v1.2.4` that I just pushed?

File: go.mod (hunk 1 above)
Comment (bhandras) [reply go.mod:208]: +1

Comment (Roasbeef) [reply go.mod:208]: Re-added!

**Pull request merged**